data/presets/presets/amenity/icecream.json

```diff
@@ -0,0 +1,21 @@
+{
+    "icon": "ice-cream",
+    "fields": [
+        "address",
+        "building_area",
+        "opening_hours",
+        "takeaway",
+        "delivery"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "ice-cream"
```

Comment:
You don't have to put "ice cream" in the terms, because it is part of the preset name and also the value.  The `terms` are more for synonyms or related words like "gelato", "sorbet", "sherbet", "frozen", "yogurt", "self serve"..